src/main/java/com/google/api/generator/spring/composer/SpringPropertiesClassComposer.java

```diff
@@ -291,8 +292,36 @@ private static MethodDefinition createGetterMethod(
 
   private static MethodDefinition createSetterMethod(
       TypeNode thisClassType, String propertyName, TypeNode returnType) {
+
+    // Common building blocks
     Variable propertyVar = Variable.builder().setName(propertyName).setType(returnType).build();
     Expr thisExpr = ValueExpr.withValue(ThisObjectValue.withType(thisClassType));
+    TypeNode oldDurationType = staticTypes.get("org.threeten.bp.Duration");
+    TypeNode newDurationType = staticTypes.get("java.time.Duration");
```

Comment:
nit: perhaps naming as `threetenDurationType` and `JavaDurationType`?

Reply:
Yup - updated to `threetenBpDurationType` and `javaTimeDurationType`.